internal/test_cases/file_search_test_case.go

```diff
@@ -0,0 +1,81 @@
+package test_cases
+
+import (
+	"fmt"
+	"path"
+	"sort"
+	"strings"
+
+	"github.com/codecrafters-io/tester-utils/test_case_harness"
+)
+
+type FileSearchTestCase struct {
+	Pattern          string
+	FilePaths        []string
+	ExpectedExitCode int
+	ExpectedOutput   []string
+	Recursive        bool
```

Comment:
```suggestion
	ShouldEnableRecursiveFlag        bool
```

Reply:
(More concrete description of what it does + more obvious that the value is a boolean)